tests3/sqlservertests.py

```diff
@@ -1374,15 +1374,16 @@ def test_cursor_messages_with_print(self):
         brand_new_cursor = self.cnxn.cursor()
         self.assertIsNone(brand_new_cursor.messages)
 
-        self.cursor.execute("PRINT 'hello world'")
-        self.assertTrue(type(self.cursor.messages) is list)
-        self.assertEqual(len(self.cursor.messages), 1)
-        self.assertTrue(type(self.cursor.messages[0]) is tuple)
-        self.assertEqual(len(self.cursor.messages[0]), 2)
-        self.assertTrue(type(self.cursor.messages[0][0]) is str)
-        self.assertTrue(type(self.cursor.messages[0][1]) is str)
-        self.assertEqual('[01000] (0)', self.cursor.messages[0][0])
-        self.assertTrue(self.cursor.messages[0][1].endswith('hello world'))
+        for message in ('hello world', 'A' * 30000):
```

Comment:
SQL Server PRINT statements will never return more than 8000 characters so this test should not try to test any value longer than that.  Bear in mind the message is prefixed with the description of the driver, etc. (e,g, `[Microsoft][ODBC Driver 17 for SQL Server][SQL Server]`) so exactly 8000 won't work either.  Perhaps use 7000 instead of 30000?

https://docs.microsoft.com/en-us/sql/t-sql/language-elements/print-transact-sql?view=sql-server-ver15#remarks

Ditto for `tests2/sqlservertests.py`.